Update logs-config.md - structured_metadata (#7026)
grafana/agent: docs/sources/static/configuration/logs-config.md

configured, except deprecated fields have been removed and the server_config is
not supported.

## structured_metadata

> Keep in mind that Grafana Agent in a deployment like this one:
> 
> ```
> promtail --> grafana-agent --> Loki
> ```
> is not able to forward `structured_metadata` received from `promtail` to `loki`.

Reviewer: ```suggestion
A Grafana Agent deployment similar to `promtail --> grafana-agent --> Loki` can't forward `structured_metadata` received from Promtail to Loki.
```

It's not perfect but it's good enough for Agent docs.

Refer to the
[Promtail documentation](/docs/loki/latest/clients/promtail/configuration/#clients)
for the supported values for these fields.

> 
> 
> 
> 

Reviewer: ```suggestion

```

Remove extra CR/LFs
